scripts/generate-docs.sh

```diff
@@ -18,6 +14,18 @@ fi
 
 tfplugindocs
 
+security_monitoring_doc_files=(
+  'docs/data-sources/security_monitoring_rules.md'
+  'docs/resources/security_monitoring_default_rule.md'
+  'docs/resources/security_monitoring_rule.md'
+)
+
+# Special formatting for security monitoring documentation files
+for file in "${security_monitoring_doc_files[@]}"
+do
+  perl -i -pe 's/Read-Only/Read-only/g' $file
+  perl -i -pe 's/Nested Schema/Nested schema/g' $file
+done
```

Comment:
Let's remove this. We can align all the docs later by updating tfplugindocs.

Reply:
But do we need to update all the docs? I did this because I thought we only wanted this special formatting for security monitoring files. In any case, I'm fine with anything 👍

Reply:
No we don't want to special case it.